test/new-e2e/pkg/components/remotehost.go

```diff
@@ -221,3 +233,16 @@ func (h *RemoteHost) buildEnvVariables(command string, envVar EnvVar) string {
 	}
 	return cmd
 }
+
+// convertToForwardSlash replaces backslashes in the path with forward slashes for Windows remote hosts.
+// The path is unchanged for non-Windows remote hosts.
+//
+// This is necessary for remote paths because the sftp package only supports forward slashes, regardless of the local OS.
+// The Windows SSH implementation does this conversion, too. Though we have an advantage in that we can check the OSFamily.
+// https://github.com/PowerShell/openssh-portable/blob/59aba65cf2e2f423c09d12ad825c3b32a11f408f/scp.c#L636-L650
+func (h *RemoteHost) convertToForwardSlash(path string) string {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
This function is not doing what it says: it is replacing `\\` with `/` only on windows. What happens if the target is Linux and the caller used backward slashes ?

Reply:
Linux paths already have forward slash path separators. backslashes are a valid Linux filename character and we have no reason to change that here.

Reply:
renamed function as discussed.